index.html

```diff
@@ -0,0 +1,637 @@
+              <p>Congressman Attended NRA Fundraiser After Skipping COVID-19 Relief Vote</p>
+            </div>
+            <hr>
+            <div class="congress1 d-flex justify-content-center">
+              <img src="https://d.newsweek.com/en/full/1574927/candle.jpg?w=95&h=95&f=9041fb7e8d050adba66d063a53b19abc" alt="">
+              <p>My Family Gave Up Sitting Shiva to Fight Coronavirus</p>
+            </div>
+            <div class="img">
+              <img src="img/21.png" alt="">
+            </div>
+            <div class="img">
+              <img src="img/22.png" alt="">
+            </div>
+            <div class="img">
+              <img src="img/23.png" alt="">
+            </div>
+            <div class="img">
+              <img src="img/24.png" alt="">
+            </div>
+
+          </div>
+        </div>
+    </section>
+
+    <section class="magazine">
+      <div class="container">
+        <div class="row n-magazine">
+          <div class="mag-heading">
+            <h4> <img src="images/l.png" alt=""> IN THE MAGAZINE</h4>
+            <hr>
+          </div>
+
+          <div class="col-lg-3  mag col-md-3 ">
+            <img src="https://d.newsweek.com/en/full/1574205/march-20-27-2020-cover.jpg?w=360&h=480&q=90&f=f32c43b8e7ecd7514c13923e4edef489" alt="">
+          </div>
+          <div class="col-lg-3 mag-1 col-md-3">
+            <img src="images/e.png" alt="">
+            <a href="#">Is Bernie Sanders Right About Medicare For All?</a>
+          </div>
+          <div class="col-lg-3 mag-1 col-md-3">
+            <img src="images/f.png" alt="">
+            <a href="#">How to Stay Sane In a Crazy Market</a>
+          </div>
+          <div class="col-lg-3 mag-1 col-md-3">
+            <img src="images/g.png" alt="">
+            <a href="#">Sanders Is Beating Biden In the Race for Donations From Hollywood's Elite</a>
+          </div>
+          <div class="col-lg-3 mag-1 col-md-3">
+            <img src="images/h.png" alt="">
+            <a href="#">Past Pandemics By The Numbers and How Coronavirus Fits In</a>
+          </div>
+          <div class="col-lg-3 mag-1 col-md-3">
+            <img src="images/i.png" alt="">
+            <a href="#">The Grandest Olympic Stadiums of All Time</a>
+          </div>
+          <div class="col-lg-3 mag-1 col-md-3">
+            <img src="images/j.png" alt="">
+            <a href="#">Project Runway's Elaine Welteroth On How Fashion Can Be a Force for Change</a>
+          </div>
+          <div class="col-lg-3 mag-1 col-md-3">
+            <img src="images/k.png" alt="">
+            <a href="#">THE AFTERMATH</a>
+          </div>
+
+        </div>
+
+      </div>
+   </div>
+    </section>
+
+    <section class="editors">
+      <h1>hi</h1>
+      <div class="container">
+        <div class="row">
+          <div class="editors-heading">
```

Comment:
- [ ] In a small screen, the elements of this section are not displaying correctly and the images look cramped. Please check your code and make sure that we can see the titles and the space between elements.
![Screenshot from 2020-03-20 12-37-56](https://user-images.githubusercontent.com/51926937/77195773-c9721100-6aa7-11ea-986c-0c9e5c082bf8.png)